samples/snapshot/pom.xml

```diff
@@ -23,13 +23,14 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
-  <!-- {x-version-update-start::current} -->
+  <!-- {x-version-update-start:google-cloud-datastore:current} -->
   <dependencies>
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-datastore</artifactId>
       <version>2.17.0</version>
```

Comment:
this is a mistake, this should be 2.17.1-SNAPSHOT (snapshot dir should always run latest). Mind updating?

Reply:
this version is particularly synced from the `main` [branch itself](https://github.com/googleapis/java-datastore/blob/main/samples/snapshot/pom.xml#L28-L32). there is no PR from renovate bot on main branch updating it to `2.17.1-SNAPSHOT`

Also looking at the history of this file, seems like it is always been using latest **non snapshot** version.

Reply:
yes, this is an error. It's always supposed to be on SNAPSHOT so that the samples are tested against main (released version is tested in this pom: https://github.com/googleapis/java-datastore/blob/main/samples/install-without-bom/pom.xml). I had to fix this earlier to get the PR working (before we cut 2.17.0):
failing: https://github.com/googleapis/java-datastore/pull/1101/commits/8692c7d9c53a86d3ef4df45d42f51ad97703a45b
fix: https://github.com/googleapis/java-datastore/pull/1101/commits/49a3a65fce6f5ce43c43a7b524f5327d9d1c57e0

Here's the correct setup as shown in the Bigtable client: https://github.com/googleapis/java-bigtable/blob/main/samples/snapshot/pom.xml

Snapshot versions will get updated as part of each SNAPSHOT release PR.

Reply:
got it, have addressed it in 48e7b49.